gamestonk_terminal/stocks/stocks_helper.py

```diff
@@ -407,22 +426,18 @@ def load(
         return [s_ticker, s_start, s_interval, df_stock]
 
 
-def display_candle(s_ticker: str, s_start: datetime, plotly: bool):
-    """Shows candle plot of loaded ticker
+def display_candle(s_ticker: str, df_stock: pd.DataFrame, plotly: bool):
```

Comment:
Also should we name poorly to something more intuitive like `interactive` (or `ion` or `it` like docker)

Reply:
Will do that